src/main/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/CreateTaskController.java

```diff
@@ -27,21 +28,18 @@
         @ApiResponse(code = 201, message = "A new task has been created for the transition"),
         @ApiResponse(code = 204, message = "No new task was created for the transition")
     })
-    public ResponseEntity createTask(@RequestBody CreateTaskRequest createTaskRequest) {
+    public ResponseEntity<Object> createTask(@RequestBody CreateTaskRequest createTaskRequest) {
         if (taskService.createTask(
             createTaskRequest.getServiceDetails(),
             createTaskRequest.getTransition(),
             createTaskRequest.getCaseId(),
             createTaskRequest.getDueDate()
         )) {
             return created(null).build();
```

Comment:
We should probably return a url to the task management api now with the get task endpoint but that should be another a story and not urgent.

Reply:
yes, it's not defined yet. Let's leave for later on, then.